src/main/java/seedu/address/logic/commands/ViewCommand.java

```diff
@@ -34,15 +35,53 @@ public ViewCommand(String tele) {
         this.tele = tele.toLowerCase();
     }
 
+    /**
+     * Checks if char array given contains any illegal characters (not letters, numbers or underscore)
+     * @param arr char array
+     * @return boolean value
+     */
+    public boolean containsIllegalCharacters(char[] arr) {
+        char[] legalCharacters = new char[]{'a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i',
+            'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x',
+            'y', 'z', '1', '2', '3', '4', '5', '6', '7', '8', '9', '0', '_'};
+        int illegalCount = 0;
+        for (Character c : arr) {
+            boolean isLegal = false;
+            for (Character lc : legalCharacters) {
+                if (!isLegal) {
+                    isLegal = c.equals(lc);
+                }
+            }
+            illegalCount = !isLegal ? illegalCount + 1 : illegalCount;
+        }
+        return illegalCount > 0;
+    }
+
+    /**
+     * Checks if telegram handle for this ViewCommand instance is at least 5
+     * characters long and that it contains no illegal characters. Otherwise
+     * throws relevant exception
+     * @throws CommandException
+     */
+    public void isValidHandle() throws CommandException {
+        char[] arr = this.tele.toCharArray();
+        if (arr.length < 5) {
```

Comment:
Change this to make use of the public static `isValidTelegramLength` method in the Telegram class (this can only be done after PR #157 is merged in)

Reply:
Now that #157 is merged in you can use the `isValidTelegramLength(String)` without needing to convert the tele string to array